### PR TITLE
api: add get region count interface

### PR DESCRIPTION
**Pull request by 3pointer**
### What problem does this PR solve? 
Add a get region count interface for http api

### What is changed and how it works?
use basic_cluster `GetRegionCount`

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)

Code changes

 - Has HTTP API interfaces change

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to be included in the release notes

### CODE REVIEW

**Comment by disksing** [comment]
Hi @3pointer Thanks for your excellent PR! Please update `server/api/api.raml` and `docs/api.html` (use `scripts/build-api.sh`)

**Comment by 3pointer** [comment]
> Hi @3pointer Thanks for your excellent PR! Please update `server/api/api.raml` and `docs/api.html` (use `scripts/build-api.sh`)

@disksing sure, I update `server/api/api.raml` and run the `scripts/build-api.sh`, please have a look, thx

File: server/api/api.raml [hunk 1]
```diff
@@ -870,6 +870,16 @@ types:
...
       500:
         description: PD server failed to proceed the request.
+  /count:
+    get:
+      description: Get region count in the cluster.
+      responses:
+        200:
+          body:
+            application/json:
+              type: Regions
```
Comment (disksing) [inline server/api/api.raml]: I think type should be `integer` or `number`.

Comment (disksing) [reply server/api/api.raml]: oh I see. It does return `Regions` but with no region inside. LGTM.

**Comment by overvenus** [comment]
@3pointer Please cherry-pick to release-3.1.

**Pull request merged**